### ISSUE TITLE
Convergence issues during training Phase One

### ISSUE THREAD

**Issue by apple-love** [CRIPAC-DIG/tgm-dlm#2]
Hello, I encountered convergence issues during training Phase One. I have looked at the details in readme, but despite several attempts, I still haven't achieved convergence. Should I modify 
https://github.com/CRIPAC-DIG/tgm-dlm/blob/main/improved-diffusion/scripts/train.py:177 
by changing the seed number, and then continue trying?

**Comment by Deno-V** [comment]
Someone has reported this to me last month. And it is confirmed that changing seeds should solve this problem. However it may need some luck… There has been another person contacting me with this problem, he did achieve good convergence after changing seeds, however he tried many times. 

This is not what I expect, this code is released after several months before I finishing this work and when I am working on this I didn’t encounter this problem. Here are some suggestions:
1. You can contact me through the email from the paper gonghaisong2021@ia.ac.cn. I can give you some codes from the older version, it should converge well.
2.  Or  You can change seed in this code, however it needs luck.
3. In the future I will fix this problem :)

**Comment by Deno-V** [comment]
After lowering the learning rate, the convergence problem is solved. (Code Updated)
